public/data/pumps.geojson

```diff
@@ -12,7 +12,7 @@
 { "type": "Feature", "properties": { "id": 705853530, "addr:full": "Witzlebenplatz 2", "image": "File:Charlottenburg Witzlebenplatz 1 Wasserpumpe-002.jpg", "pump:style": "Lauchhammer", "check_date": "unbekannt", "pump:status": "unbekannt" }, "geometry": { "type": "Point", "coordinates": [ 13.2933791, 52.5090859 ] } },
 { "type": "Feature", "properties": { "id": 705853538, "addr:full": "Fritschestraße", "image": "File:Charlottenburg Fritschestraße 45 Wasserpumpe-001.jpg", "pump:style": "Rümmler", "check_date": "unbekannt", "pump:status": "unbekannt" }, "geometry": { "type": "Point", "coordinates": [ 13.3005912, 52.5077142 ] } },
 { "type": "Feature", "properties": { "id": 705853544, "addr:full": "Stuttgarter Platz", "image": "File:Charlottenburg Stuttgarter Platz Wasserpumpe-001.jpg", "pump:style": "Rümmler", "check_date": "unbekannt", "pump:status": "unbekannt" }, "geometry": { "type": "Point", "coordinates": [ 13.3004354, 52.5048993 ] } },
-{ "type": "Feature", "properties": { "id": 712701215, "addr:full": "Schillerstraße 120", "image": "File:Charlottenburg Schillerstraße 120 Wasserpumpe.jpg", "pump:style": "Rümmler", "check_date": "unbekannt", "pump:status": "unbekannt" }, "geometry": { "type": "Point", "coordinates": [ 13.3188987, 52.5109317 ] } },
+{ "type": "Feature", "properties": { "id": 712701215, "addr:full": "Schillerstraße 120", "image": "File:Charlottenburg Schillerstraße 120 Wasserpumpe.jpg", "pump:style": "Borsig", "check_date": "unbekannt", "pump:status": "unbekannt" }, "geometry": { "type": "Point", "coordinates": [ 13.3188987, 52.5109317 ] } },
```

Comment:
Why where the pumps GeoJSON changed here?

Reply:
that's weird, i didn't touch em